external/anomaly/tasks/inference.py

```diff
@@ -294,8 +294,9 @@ def save_model(self, output_model: ModelEntity) -> None:
     def _set_metadata(self, output_model: ModelEntity):
         output_model.set_data("image_threshold", self.model.image_threshold.value.cpu().numpy().tobytes())
         output_model.set_data("pixel_threshold", self.model.pixel_threshold.value.cpu().numpy().tobytes())
-        output_model.set_data("min", self.model.min_max.min.cpu().numpy().tobytes())
-        output_model.set_data("max", self.model.min_max.max.cpu().numpy().tobytes())
+        if hasattr(self.model, "normalization_metrics") and self.model.normalization_metrics.state_dict() is not None:
+            for key, value in self.model.normalization_metrics.state_dict().items():
```

Comment:
I guess this would work, but the openvino task assumes that the output model contains the "min" and "max" keys specifically. So this would not allow generalizing to other normalization methods.

Reply:
Ah right! I think I'll revert it just for consistency.

Reply:
I think that would be safer